SDCTA/src/components/Body.tsx

```diff
@@ -0,0 +1,56 @@
+import landscape from '../Images/unsplash_dsV4yvL5UJk.jpg';
+import EdVis from '../Images/Graph.jpg';
+import './body.css'
+import EducationVis from '../Images/Education_Standin.png';
+import HomelessnessVis from '../Images/Homelessness_Standin.png';
+import MunicipalVis from '../Images/Municipal_Standin.png'
+
+
+export const Body = () => {
+	return (
+		<><div>
+			<div className="heading" >
+				SDTEF Data Museum
+			</div>
+			<div className="description" >
+				Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+			</div>
+			<img src={landscape} alt="sd landscape" />
+		</div>
+			<div className="box" >
```

Comment:
Only issue I see is maybe more descriptive classnames than "box" or "one", something more along the lines of "preview_box" and "first_visual"